server/routes/remandRoutes.ts

```diff
@@ -313,17 +316,45 @@ export default class RemandRoutes {
     const { nomsId } = req.params
     const { username } = res.locals.user
     const { bookingId } = res.locals.prisoner
-    const adjustments = (await this.adjustmentsService.findByPersonOutsideSentenceEnvelope(nomsId, username)).filter(
-      it => it.adjustmentType === 'REMAND',
-    )
+    const adjustments = await this.adjustmentsService.findByPersonOutsideSentenceEnvelope(nomsId, username)
+
     const sentencesAndOffences = await this.prisonerService.getSentencesAndOffencesFilteredForRemand(
       bookingId,
       username,
     )
     this.adjustmentsStoreService.clear(req, nomsId)
 
+    const unusedDeductionType = await this.unusedDeductionsService.getCalculatedUnusedDeductionsMessage(
+      nomsId,
+      bookingId,
+      false,
+      username,
+    )
+
+    let unusedDeductionMessage = null
+    if (unusedDeductionType === 'NOMIS_ADJUSTMENT' && config.featureToggles.reviewUnusedDeductions) {
```

Comment:
Again this is so much duplication, can you pull everything unused deductions related into a separate view model. Then share that model between the hub, remand and tagged-bail.